pages/test_analytics/test_ownership.md

```diff
@@ -0,0 +1,117 @@
+
+## Example TESTOWNER file
+
+```bash
+# This is a comment.
+# Each line is a file pattern followed by one or more team slugs.
+
+# These teams will be the owners for everything in
+# the repo. While both teams own the test, the first team listed will
+# become the default owner for tests in this directory. Unless
+# a later match takes precedence, team-slug-1 will have any
+# flaky tests assigned to them by default.
+*       team-slug-1 team-slug-2
+
+# Order is important; the last matching pattern takes the most
+# precedence. Any test file ending with _spec.rb will be assigned
+# to the test-analytics team and not team-slug-1.
+*_spec.rb    test-analytics #This is an inline comment.
+
+# Only Buildkite teams can be specified as test owners. Teams should
+# be identified by their slug. Teams must have
+# explicit access to the suite the test belongs to. In this example,
+# the pipelines team owns all .rb files.
+*.rb pipelines
+
+# In this example, packages owns any files in the spec/packages/
+# directory at the root of the test directory and any of its
+# subdirectories.
+/spec/packages/ packages
+
+# The `spec/features/*` pattern will match files like
+# `spec/features/application_spec.rb` but not further nested files like
+# `spec/features/pipelines/application_spec.rb`.
+spec/features/*  test-analytics
+
+# In this example, pipelines owns any file in a pipelines directory
+# anywhere in your test directory.
+pipelines/ pipelines
+
+# In this example, packages owns any file in the `/spec/packages`
+# directory in the root of your test directory and any of its
+# subdirectories.
+/spec/packages/ packages
+
+# In this example, test-analytics owns any test in a `/analytics` directory such as
+# `/models/analytics`, `/features/analytics`, and `/models/organizations/analytics`. # Any tests in an `/analytics` directory will belong to team test-analytics.
+**/analytics test-analytics
+
+# In this example, pipelines owns any test in the `/spec`
+# directory in the root of your test directory except for the `/spec/models/packages`
+# subdirectory, as this subdirectory has its own owner packages
+/spec/ pipelines
+/spec/models/packages packages
+```
+{: codeblock-file="TESTOWNERS"}
+
+## FAQs
+
+### Can multiple suites use the same TESTOWNER file?
+Yes, there's nothing stopping you from uploading the same file to multiple suites. A suite only has one active TESTOWNER file at a time.
+
+### Can more than one team own a test?
+More than 1 team may own a test, and the order of teams in your TESTOWNER file is important. The first team listed will be the default owner, and they will be auto-assigned to the test if it flakes. Any team with suite access can override this auto-assignment.
+
+### Why aren't my teams being assigned ownership over my tests?
+A TESTOWNER file uses Buildkite team slugs instead of user names. Your team slug will be your team name in kebab-case. You can view your teams in your organization settings, or fetch them from our API:
```

Comment:
```suggestion
### Why aren't my teams being assigned ownership over my tests?

A TESTOWNER file uses Buildkite team slugs instead of user names. Your team slug will be your team name in kebab-case. You can view your teams in your organization settings, or fetch them from our API:
```